pkg/utils/ffmpeg.go

```diff
@@ -0,0 +1,25 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func ffmpegCommand(args []string) (string, error) {
+	cmd := exec.Command("ffmpeg", args...) // Constrain this to ffmpeg to permit security scanner to see that the command is safe.
+	cmd.Env = os.Environ()
+	out, err := cmd.CombinedOutput()
+	return string(out), err
+}
+
+// AudioToWav converts audio to wav for transcribe.
+// TODO: use https://github.com/mccoyst/ogg?
+func AudioToWav(src, dst string) error {
```

Comment:
mudler: I pulled this out here since it's extremely likely that I'll need it in SoundGeneration as well when I get around to exposing audio-conditioned generation.